README.md

```diff
@@ -33,6 +33,16 @@ inherit_gem:
   rubocop-aha: lib/config.yml
 ```
 
+## Semantic Versioning
+
+MAJOR version is currently 0: this is unspecified.
+
+MINOR version will update for breaking changes to rubocop, e.g. when existing
+rules break compatibility.
```

Comment:
This struck me as kind of weird, but I guess it's consistent with the way that rubocop itself implements semver (according to the discussion in https://github.com/rubocop-hq/rubocop/issues/4243#issuecomment-291751099, it's in such a state of flux that the major version will be pinned to `0` for the foreseeable future).